src/lib/OTA/OTA.h

```diff
@@ -64,7 +64,9 @@ typedef struct {
         } PACKED dbg_linkstats;
         /** PACKET_TYPE_MSP **/
         struct {
-            uint8_t packageIndex;
+            uint8_t packageIndex:5,
```

Comment:
If you change the size of `packageIndex`, you will need to also change `ELRS4_TELEMETRY_SHIFT` and `ELRS8_TELEMETRY_SHIFT` in `telemetry_protocol.h` appropriately. These values shift 255 down by a number if bites to give the correct value for `ELRS4_TELEMETRY_MAX_PACKAGES` and `ELRS8_TELEMETRY_MAX_PACKAGES` respectively.

Reply:
I think Jye is gunning for adding a new OTA struct that will avoid a breaking version change when this finally merges.
The TX and RX both know when they are in mavlink mode, so they will use the new OTA struct.
This should mean we can revert these changes.

Reply:
That is probably the best way to go.